### ISSUE TITLE
Mod not working. (game version  1.0.7 Steam)

### ISSUE THREAD

**Issue by sanovskiy** [caioreix/BetterMissions#15]
**Game Version:** 1.0.7.80689b4  
**Mod Framework:** BepInExPack_V_Rising v1.691.3  
**Mod Used:** BetterMissions v2.0.1  

**Description:**
After loading the game with the BetterMissions mod, mission durations do not reflect the changes specified in the mod settings.

**Steps to Reproduce:**
1. Set the mission duration for all missions to 300 seconds (5 minutes) in the mod settings.
2. Start the game with BepInExPack and BetterMissions mod installed.
3. Start any mission.
4. Observe the mission timers.

**Expected Result:**
All mission durations should be 5 minutes long.

**Actual Result:**
Both the planning and running mission timers display the default (vanilla) values instead of the modified 5-minute duration.

**Additional Information:**
BepInEx shows that the mod is loaded correctly at the start of the game.

**Comment by Piknim** [comment]
I am having the same issue. The mission timer settings are note affecting length and it seems that injury settings are also not working. I intentionally put the injury chance to zero for all mission types and sent a bunch of servants on 2 hour mission and they got injured anyway.

**Comment by caioreix** [comment]
Ill check this issue asap

**Comment by GregariousJB** [comment]
Yeah, I came here to check if anyone had reported this. For me it's the Injury setting - I set all the injury chances to 0 but one of my servants came back injured. 

Will changing the injury chance in the config also update the in-game UI while selecting a mission? I assumed because of your feature listed in the description: "Real mission duration displayed on the HUD" that I might still see "15%" or "20%" injury chance even though it's actually 0%.

**Comment by Yourname942** [comment]
I am also not seeing the timers affect the in game settings either.. (It is still showing the default values) I am using the thunderstore app, and made sure to edit both the server and client configs..

**Comment by caioreix** [comment]
Sorry for the delay in responding. At the moment, I'm a bit busy with other projects and work. As soon as I have some free time, I'll update the mod and fix these config errors.

**Comment by nixmomo** [comment]
Still the same issue some months later... merry christmas :D

**Comment by caioreix** [comment]
https://github.com/caioreix/BetterMissions/pull/20
